jest/jest-replace-attribute-snapshot-serializer.js

```diff
@@ -1,17 +1,14 @@
 const DEFAULT_UUID = 'aaaabbbb-cccc-dddd-eeee-ffffffffffff'
 
-function hasUuidAttribute(element: Element, attribute: string): boolean {
+function hasUuidAttribute(element, attribute) {
```

Comment:
**Comment**: I turned this file into a plain JS file because it doesn’t look like `@jest/globals` exports the `SnapshotSerializerPlugin` type.

Reply:
Wondered if you could you import from `pretty-format` instead?

https://github.com/facebook/jest/blob/main/packages/pretty-format/src/types.ts#L69-L98

Reply:
Good idea. That’s as good as it’s gonna get. Done.